### PR TITLE
chore(examples): update kaniko example to use action configs

### LINKED ISSUE #3801: 0.13 example update: kaniko

### PR DESCRIPTION
**Pull request by Walther**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #3801

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/kaniko/README.md [hunk 1]
```diff
@@ -11,10 +11,16 @@ name: kaniko
 environments:
   - name: local
   - name: remote
-    defaultNamespace: ${var.userId}
 providers:
-  - name: local-kubernetes
-    environments: [local]
```
Comment (edvald) [inline examples/kaniko/README.md:17]: Is there a reason to move this? If there is, best to remove the `local` environment above as well.

File: examples/kaniko/frontend/garden.yml [hunk 2]
```diff
@@ -1,27 +1,43 @@
...
+    - path: /call-backend
+      port: http
+
+---
+kind: Test
+type: container
+build: frontend
+name: frontend-unit
+spec:
+  args: [npm, test]
```
Comment (edvald) [inline examples/kaniko/frontend/garden.yml:33]: Suggest using `command` here instead, seeing as we'd never want this as arguments to an `ENTRYPOINT` defined in the Dockerfile.

File: examples/kaniko/README.md (hunk 1 above)
Comment (vvagaytsev) [reply examples/kaniko/README.md:17]: Ah, yes, actually, we can leave this, just to emphasize that the `local-kubernetes` provider does not use kaniko build. Fixed in the force-pushed commit.

File: examples/kaniko/frontend/garden.yml (hunk 2 above)
Comment (vvagaytsev) [reply examples/kaniko/frontend/garden.yml]: Fixed in https://github.com/garden-io/garden/pull/3803/commits/200844d72c10579039ec7dea4ccfa160bf08cd8e

File: examples/kaniko/README.md (hunk 1 above)
Comment (vvagaytsev) [reply examples/kaniko/README.md:17]: Fixed in https://github.com/garden-io/garden/pull/3803/commits/f6699bc78b61c0c0e97bd8c9af2e36a9bd8c12bc

**Review by edvald — APPROVED** [review]

**Pull request merged**